apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -186,29 +187,19 @@ export function ActionSection(): JSX.Element {
                         )}
                     </div>
                 )}
-
-                {(mintWithCrossmint.error != "") && (
-                    <div>
-                        <BaseCallout extraClasses={{ calloutWrapper: "md:h-[85px] h-[109px] mt-[12px]", calloutFront: "!justify-start" }} isWarning>
-                            <div className="flex md:gap-[21px] gap-[10px]">
-                                <span className="block mt-2"><WarningIcon /></span>
-                                <div>
-                                    <span className="block font-bold text-lg">Error minting with Credit/Debit Card</span>
-                                    {/* We currently have no way of knowing all the possible errors that could come from minting with crossmint, so we should log in the console and display a generic error rather than the error message. */}
-                                    <span className="block font-medium text-lg">There was an error processing your credit card payment</span>
-                                </div>
-                            </div>
-                        </BaseCallout>
-                    </div>
-                )}
             </div>
```

Comment:
Do we never show an error ?

Reply:
The errors are displayed in side the Crossmint modal. This error would have been displayed if we were managing the Crossmint order lifecycle outside of embedded checkout.